### DISCUSSION TITLE
how to add a new device to the list /supported-devices

### DISCUSSION THREAD

**Discussion by AndrewVasilyeff** [Koenkk/zigbee2mqtt#25265] [General]
Could you please tell me how to add a new device to the list https://www.zigbee2mqtt.io/supported-devices/ We are the equipment supplier.

**Comment by lux73** [comment]
https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

**Comment by AndrewVasilyeff** [comment]
Thank you. I know how to connect the device. The question is how to add the device to the list - https://www.zigbee2mqtt.io/supported-devices/

**Comment by MSL-DA** [comment]
<img width="969" alt="Skærmbillede 2024-12-19 kl  21 55 39" src="https://github.com/user-attachments/assets/d8de40ea-be6c-4969-8e52-78dba17b72f8" />

**Reply by AndrewVasilyeff** [reply]
Thank you !!!
